#201 lucacomments

baxgas opened the pull request:
Here are the changes from the feedback
? Should I put the cover image in the stream player as in the Prog mcs comments?
? I exchanged the logo to the comic version in the header, but should it also be replaced in the other places where the logo is the placeholder?

gammaw reviewed (changes requested):
Nice changes @baxgas! The fixes, new features such as the seeker, etc. 

But that's a lot of things in one big PR, things that are different in nature: fixes (such as Soundcloud -> Mixcloud), agreed changes (such as new Lahmacun logo), and proposed changes (such as hiding the sand clock). 

I would have thematic PRs, even if we'll have many PRs. For example, I'm obviously good with the fixes and with the approved changes, but I'm not sure about new naming (arcsive) and reshaping the player at this change (I added two inline comments), and now these changes "block" one another. 

How about spitting this PR into smaller ones? I think it should be easy by cherry picking commits into new branches. I can try to help if you don't manage.

gygabor commented on app/components/HeadNavigation.vue:
```diff
@@ -11,9 +11,9 @@
-          Archive
+      <li :class="(baseRoute === '/arcsive/' || (baseRoute === '/shows/' && routeArray.length === 4)) && 'selected'">
+        <NuxtLink to="/arcsive/">
+          Arcsive
```
I don't think if `Arcsi` or `Archive` is the good name for the `Archive`. I'm sure a Sarcófago fan from Belém would have no idea what it means. Or a Norwegian trap fanatic. Actually nobody except us.

baxgas commented:
> Nice changes @baxgas! The fixes, new features such as the seeker, etc.
> 
> But that's a lot of things in one big PR, things that are different in nature: fixes (such as Soundcloud -> Mixcloud), agreed changes (such as new Lahmacun logo), and proposed changes (such as hiding the sand clock).
> 
> I would have thematic PRs, even if we'll have many PRs. For example, I'm obviously good with the fixes and with the approved changes, but I'm not sure about new naming (arcsive) and reshaping the player at this change (I added two inline comments), and now these changes "block" one another.
> 
> How about spitting this PR into smaller ones? I think it should be easy by cherry picking commits into new branches. I can try to help if you don't manage.

I added back the sandclock, and renamed the Arcsives. also added double click for faster seeking
Sure, can be more issues, but if it is ok, we could close%merge this and start new ones.

gammaw reviewed (approved):
Thanks!
